deployment/terraform/create.go

```diff
@@ -407,3 +410,26 @@ func (t *Terraform) getOutput() (*terraformOutput, error) {
 	}
 	return &output, nil
 }
+
+func pingServer(addr string) error {
+	mlog.Info("Checking server status:", mlog.String("host", addr))
+	client := model.NewAPIv4Client(addr)
+	client.HttpClient.Timeout = 10 * time.Second
+	timeout := time.After(30 * time.Second)
+
+	for {
+		select {
+		case <-timeout:
+			return fmt.Errorf("timeout after 30 seconds, server is not responding")
+		case <-time.After(3 * time.Second):
+			_, resp := client.GetPingWithServerStatus()
+			if resp.Error != nil {
+				mlog.Debug("got error", mlog.Err(resp.Error))
+				mlog.Info("Waiting for the server...")
+				continue
+			}
+			mlog.Info("Server status is OK")
```

Comment:
I thought we were going to check for the string that comes with the response now that the file issue is taken care of ?

Reply:
We can, but it already returns an error if the server status is not OK. Basically, checking the error and string is the same 😅

Reply:
Resolved offline.